src/overlays/actors/ovl_En_Syateki_Okuta/z_en_syateki_okuta.c

```diff
@@ -76,15 +85,15 @@ static InitChainEntry sInitChain[] = {
     ICHAIN_F32(targetArrowOffset, 6500, ICHAIN_STOP),
 };
 
-Color_RGBA8 D_80A37B90 = { 255, 255, 255, 255 };
+Color_RGBA8 sDustPrimColor = { 255, 255, 255, 255 };
```

Comment:
If you are going to use pretty generic names, should probably make them static now.